kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -125,19 +134,48 @@ class ResidualBasedBlockBuilderAndSolver
         // Validate default parameters
         Parameters default_parameters = Parameters(R"(
         {
-            "name" : "ResidualBasedBlockBuilderAndSolver"
+            "name"                               : "ResidualBasedBlockBuilderAndSolver",
+            "block_builder"                      : true,

```

Comment:
```suggestion

```

I think this should not be here

Reply:
If I remove it could raise an error for the python solver changes

Reply:
Hm for this PR it would not matter, then we need to find a better solution when we adapt the python solver.
IMO we should not add such things just to be backwardcompatible, since we have to change things in any case
Just my opinion, not a blocker

Reply:
It is backward compatible, check the modified code in the solver

Reply:
It is not, we have to move the setting `block_builder` inside the `b&s_settings`
Anyway, discussion for another PR

Reply:
OK, the change is applied in another PR (#5939 ), but this will avoid to modify the block B&S in that PR. Sorry for the inconvenience